### PR TITLE
cinder: add filter for volume_backend_name

### PR DESCRIPTION
**Pull request by auhlig**
Checklist:

- [x] If this PR is about a plugin, I tested the plugin against an OpenStack cluster.
- [x] I updated the documentation to describe the semantical or interface changes I introduced.

### CODE REVIEW

File: pkg/plugins/capacity_cinder.go [hunk 1]
```diff
@@ -54,18 +54,25 @@ func (p *capacityCinderPlugin) Scrape(driver limes.Driver) (map[string]map[strin
 	}
 
 	var result gophercloud.Result
+	var volumeBackendName string
+
+	if p.cfg.Cinder.VolumeBackendName != "" {
+		volumeBackendName = p.cfg.Cinder.VolumeBackendName
+	}
```
Comment (majewsky) [inline pkg/plugins/capacity_cinder.go]: Or just `volumeBackendName := p.cfg.Cinder.VolumeBackendName`.

File: pkg/plugins/capacity_cinder.go [hunk 2]
```diff
@@ -97,7 +104,11 @@ func (p *capacityCinderPlugin) Scrape(driver limes.Driver) (map[string]map[strin
 
 	//add results from scheduler-stats
 	for _, element := range limitData.Pools {
-		totalCapacity += uint64(element.Capabilities.TotalCapacity)
+		if (volumeBackendName != "") && (element.Capabilities.VolumeBackendName == volumeBackendName) {
+			totalCapacity += uint64(element.Capabilities.TotalCapacity)
+			util.LogDebug("Considering %s with volume_backend_name %s", element.Name, element.Capabilities.VolumeBackendName)
+		}
+		util.LogDebug("Not considering %s with volume_backend_name %s", element.Name, element.Capabilities.VolumeBackendName)
```
Comment (majewsky) [inline pkg/plugins/capacity_cinder.go]: This skips all volume types if the `volume_backend_name` is not configured. Also, it prints the `not considering` message even when considering the volume type.

Comment (auhlig) [inline pkg/plugins/capacity_cinder.go]: I need more coffee

**Comment by auhlig** [comment]
Better now @majewsky?

**Pull request merged**